kobo/apps/mass_emails/tests/test_old_email_record_mark_as_failed.py

```diff
@@ -0,0 +1,39 @@
+from datetime import timedelta
+
+from constance.test import override_config
+from django.utils.timezone import now
+from django.test import TestCase
+
+from kobo.apps.kobo_auth.shortcuts import User
+from kobo.apps.mass_emails.models import (
+    MassEmailConfig,
+    MassEmailJob,
+    MassEmailRecord,
+    EmailStatus
+)
+from kobo.apps.mass_emails.tasks import (
+    mark_old_enqueued_mass_email_record_as_failed
+)
+
+
+class TestOldEmailRecordMarkAsFailed(TestCase):
+    @override_config(PROJECT_OWNERSHIP_INVITE_EXPIRY=7)
+    def test_mark_old_enqueued_mass_email_record_as_failed(self):
+        """
+        Test that a celery updates the status of 'enqueued' MassEmailRecord
```

Comment:
Can we add a second MassEmailRecord that is not old enough to make sure that it does not get updated?